druid/src/command.rs

```diff
@@ -382,11 +387,11 @@ impl<T: Any> Selector<T> {
     /// as `Selector<()>` implements `Into<Command>`.
     ///
     /// By default, the command will have [`Target::Auto`].
-    /// The [`Command::to`] method can be used to override this.
+    /// The [`Selector::to`] method can be used to override this.
     ///
-    /// [`Command::new`]: struct.Command.html#method.new
-    /// [`Command::to`]: struct.Command.html#method.to
-    /// [`Target::Auto`]: enum.Target.html#variant.Auto
+    /// [`Command::new`]: Command::new
+    /// [`Selector::to`]: Selector::to
+    /// [`Target::Auto`]: Target::Auto
```

Comment:
I haven't quite got the hang on intra-doc links, but I think these are redundant?

Reply:
oh yeah I think that's the case.